components_esp32s3.lock

```diff
@@ -0,0 +1,15 @@
+dependencies:
```

Comment:
we should add the .lock files produced by the component manager .gitignore?

Reply:
There will be no lock files produced by `esp-idf-svc`, as per above.